task_02/ex4.py

```diff
@@ -0,0 +1,21 @@
+"""Easy Unpack."""
+
+# https://py.checkio.org/en/mission/easy-unpack/
+
+
+def easy_unpack(tuple_arg):
+    """Get an tuple and returns a tuple with 3 elements - first, third.
+
+    And second to the last for the given array.
+    """
+    if type(tuple_arg) is not tuple:
```

Comment:
If we check input args and raise exception why don't we check that len(tuple_arg)>=3?

Reply:
I don't check it, as interpreter raises the corresonding exception by itself, whilst a list as function argument isn't considered as an error.

Reply:
We have next situation: you check if your argument is a tuple. But your will never catch this exception cause you will catch it on your `tpl = tuple(sep_list)`. It's redundant and non pythonic code here. 
When I enter tuple with 2 elements, I will see IndexError in console and I can't understand what's wrong until i look through your code. 
Don't you find it a bit strange? I do.